mapbox_vector_tile/encoder.py

```diff
@@ -56,13 +56,28 @@ def transform(shape, func):
     raise ValueError('Unknown geometry type, "%s"' % shape.type)
 
 
+def on_invalid_geometry_raise(shape):
+    raise Exception('Invalid geometry: %s' % shape.wkt)
+
+
+def on_invalid_geometry_ignore(shape):
+    return None
+
+
+def on_invalid_geometry_make_valid(shape):
+    if shape.type in ('Polygon', 'MultiPolygon'):
+        shape = shape.buffer(0)
```

Comment:
If I had to _guess_, I'd assume that it would preserve the winding order... But it's entirely possible that it doesn't, or resets it to the more usual clockwise. It probably doesn't stick to integer coordinates either. We should probably be taking the output and re-quantizing it, orienting it and re-checking its validity. :scream_cat: